exporter/datadogexporter/stats.go

```diff
@@ -26,8 +26,7 @@ const (
 )
 
 // ComputeAPMStats calculates the stats that should be submitted to APM about a given trace
-func ComputeAPMStats(tracePayload *pb.TracePayload, calculator *stats.SublayerCalculator, pushTime int64) *stats.Payload {
-
+func ComputeAPMStats(tracePayload *pb.TracePayload, calculator *sublayerCalculator, pushTime int64) *stats.Payload {
```

Comment:
Why public?

Reply:
I was trying to maintain the visibility of existing functions that were already public. I have double checked with the person who wrote this code and these and other functions can be made private, so I have done so. 

I can also make these changes in a separate PR if need be.